pkg/commands/start_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package commands
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	tcpPort = "5353"
+	udpPort = "5353"
+)
+
+func TestStart(t *testing.T) {
+	cmd := run(t, "127.0.0.1", tcpPort, udpPort, "host.rd.test=111.111.111.111,host2.rd.test=222.222.222.222")
+	defer cmd.Process.Kill()
```

Comment:
Should we use `.Signal(SIGTERM)` instead, to gracefully exit? This sends a `SIGKILL` instead.

Reply:
I would assume in the test environment `SIGKILL` would be more ideal to reduce the flakes since it can not be ignored or blocked. Also, there is a potential that `SIGTERM` may not kill the child processes. 🤷

Reply:
👍 on using `SIGKILL` for the test.